src/errorhandling/ErrorHandler.spec.ts

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable @typescript-eslint/ban-ts-comment */
-import { ApiPromise, SubmittableResult } from '@polkadot/api'
+import { SubmittableResult } from '@polkadot/api'
 import { Tuple } from '@polkadot/types'
-import { ErrorHandler } from './ErrorHandler'
+import { ErrorHandler } from '.'
```

Comment:
I think it might be better to have it as `import { ErrorHandler } from './ErrorHandler'`, since it is more verbose.

Reply:
that won't work - we only export as ErrorHandler from the index